.github/workflows/docs.yml

```diff
@@ -10,6 +10,7 @@ permissions:
   contents: read
   pages: write
   id-token: write
+  actions: read
```

Comment:
why is this needed ?

Reply:
only access added should be following ( am I missing something ? ) https://github.com/microsoft/fluentui/pull/33829/files#diff-0e99d05f4114307b51d9215a468223cb3314c669b1a8add474d5468ccb9337ebL60-L62

Reply:
@Hotell hm, I read it from in changelog note [here](https://github.com/actions/deploy-pages/releases/tag/v4.0.0). But if it works without, then I will remove it.